resource_management_mobile/angular.json

```diff
@@ -31,7 +31,7 @@
                 "output": "./svg"
               }
             ],
-            "styles": ["src/theme/variables.scss","src/theme/theme.scss", "src/global.scss"],
+            "styles": ["src/theme/variables.scss","src/theme/theme.scss", "node_modules/bootstrap/dist/css/bootstrap.min.css", "src/global.scss"],
```

Comment:
Why do we need boostraop in mobile?

Reply:
For some styling purposes, we used that.